components/atoms/CardLink/CardLinkStyles.ts

```diff
@@ -7,7 +7,7 @@ interface Props {
 export const StyledCardLink = styled.a<Props>`
     cursor: pointer;
     height: 100%;
-    max-height: 377px;
+    max-height: 100%;
```

Comment:
This seems to work down to Safari 11.1. ✅

The cards start to break on and below 10.1. ❌ I'm not sure how far we want to support but I think we should try and support as far back as possible. Here are some screenshots:

![Screen Shot 2022-01-04 at 3 28 52 PM](https://user-images.githubusercontent.com/1733936/148119870-aa4ced74-4a8c-419f-ab46-447cf5f68b40.png)

![Screen Shot 2022-01-04 at 3 29 28 PM](https://user-images.githubusercontent.com/1733936/148119940-d54331ac-ae83-4aaa-95b3-04b55fc64df2.png)